src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/rerank/validate.ts

```diff
@@ -47,13 +43,12 @@ export const validate = (
   const targetName = targetField?.name || 'rerank';
 
   // Sets the target field so the column is recognized after the command is applied
-  context?.userDefinedColumns.set(targetName, [
-    {
-      name: targetName,
-      location: targetField?.location || location,
-      type: 'keyword',
-    },
-  ]);
+  context?.columns.set(targetName, {
```

Comment:
And this

Reply:
We no longer want to use custom column names like `RERANK col0 =` ? I also need this input to remove the col0 suggestion from autocomplete

Reply:
I dont think we will with Drew's changes in this PR but maybe I am wrong. Drew keep me honest here